.github/workflows/it-tests.yml

```diff
@@ -63,8 +63,13 @@ jobs:
       PREPARE_TEST_ENV_TYPE: ${{ matrix.testEnvironment }}
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: fregante/setup-git-user@v2
```

Comment:
why do you need that ? it seems to do the same as what you do line 69-72
also this repo only has 67 stars

Reply:
don't need it, I forgot to remove it (I did the `git config` manually instead)